### PR TITLE
Functional tests - Fix random errors in the nightly 1.7.8.x

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Fix random errors in the nightly 1.7.8.x
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | no need
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/campaigns/functional/FO/09_productPage/01_quickView.js [hunk 1]
```diff
@@ -286,30 +300,44 @@ describe('Product quick view', async () => {
...
 
       await homePage.selectProductColor(page, 1, 'Black');
 
-      pageTitle = await productPage.getPageTitle(page);
+      const pageTitle = await productPage.getPageTitle(page);
       await expect(pageTitle.toUpperCase()).to.contains(firstProductData.name);
+    });
 
-      const imageSecondColor = await productPage.getProductImageUrls(page);
+    it('should product image be different from the ', async function () {
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/09_productPage/01_quickView.js]: from the `what ?`

File: tests/UI/pages/FO/home/index.js [hunk 2]
```diff
@@ -221,6 +221,15 @@ class Home extends FOBasePage {
     };
   }
 
+  /**
+   * Is quick view product modal visible
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  isQuickViewProductModalVisible(page) {
```
Comment (Progi1984) [inline tests/UI/pages/FO/home/index.js]:
```suggestion
  async isQuickViewProductModalVisible(page) {
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks

File: tests/UI/campaigns/functional/FO/09_productPage/01_quickView.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/FO/09_productPage/01_quickView.js]: done thx !

File: tests/UI/pages/FO/home/index.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/FO/home/index.js]: Done :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Pull request merged**